Sources/PostgreSQL/Connection.swift

```diff
@@ -16,12 +16,25 @@ public final class Connection {
         return false
     }
 
-    public init(host: String = "localhost", port: String = "5432", dbname: String, user: String, password: String) throws {
-        self.connection = PQconnectdb("host='\(host)' port='\(port)' dbname='\(dbname)' user='\(user)' password='\(password)'")
+    public init(params: String ...) throws {
```

Comment:
Wouldn't it be better to simply pass a `[String:String]` dictionary. This allows for invalid initialisations with an unbalanced amount of parameters. Although that mistake is quickly resolved, I think it is better to prevent it from happening in the first place.